dvc/command/stage.py

```diff
@@ -0,0 +1,198 @@
+import argparse
+import logging
+
+from dvc.command import completion
+from dvc.command.base import CmdBase, append_doc_link, fix_subparsers
+
+logger = logging.getLogger(__name__)
+
+
+class CmdStageAdd(CmdBase):
+    def run(self):
+        from dvc.stage.utils import create_stage_from_cli
+
+        kwargs = vars(self.args)
+
+        cmd = kwargs.pop("command")
+        assert len(cmd) >= 1
+        kwargs["cmd"] = cmd[0] if len(cmd) == 1 else cmd
```

Comment:
Supporting multiple commands now. But, we should have an opinion here: either a multiline string or a list of cmds.

Reply:
list sounds better to me :)